presto-docs/src/main/sphinx/release/release-0.288.rst

```diff
@@ -0,0 +1,80 @@
+* Fix incorrect behaviors when defining duplicate field names in RowType and throw exception uniformly. :pr:`22618`
+* Fix regr_r2 result incorrect. :pr:`22611`
+* Fix the latency regression for queries with large IN clause. :pr:`22661`
+* Fix wrong results when queries using materialized CTEs have multiple common filters pushed into the CTE. :pr:`22700`
+* Improve :doc:`/sql/explain-analyze` statement to support a ``format`` argument with values of ``<TEXT|JSON>``. :pr:`22733`
+* Improve README.md and CONTRIBUTING.md. :pr:`22918`
+* Improve configuring worker threads relative to core count by setting the ``task.max-worker-threads`` configuration property to ``<multiplier>C``. For example, setting the property to ``2C`` configures the worker thread pool to create up to twice as many threads as there are cores available on a machine. :pr:`22809`
+* Improve logging for RowExpressionRewriteRuleSet and StatsRecordingPlanOptimizer optimizers to include more information. :pr:`22765`
+* Improve session property ``property-use_broadcast_when_buildsize_small_probeside_unknown`` to do broadcast join when probe side size is unknown and build side estimation from HBO is small. :pr:`22681`
+* Improve the estimation stats recorded during query optimization. :pr:`22769`
+* Improve Presto C++ documentation. :pr:`22717`
+* Improve error code for cast from DOUBLE or REAL to BIGINT, INTEGER, SMALLINT or TINYINT for out of range values from NUMERIC_VALUE_OUT_OF_RANGE to INVALID_CAST_ARGUMENT. :pr:`22917`
+* Improve handling of floating point numbers in Presto to consistently treat NaNs as larger than any other number and equal to itself. It also changes the handling of positive and negative zero to always be considered equal to each other. Read more here: https://github.com/prestodb/rfcs/blob/main/RFC-0001-nan-definition.md. The new nan behavior can be disabled by setting the configuration property use-new-nan-definition to false. This configuration property is intended to be temporary to ease migration in the short term, and will be removed in a future release. :pr:`22386`
+* Add HBO for CTE materialized query. :pr:`22606`
+* Add Prestissimo support for CTAS into bucketed (but not partitioned) tables. :pr:`22737`
+* Add :doc:`/presto_cpp/properties` documentation. :pr:`22885`
+* Add PR number to the release note entry examples in pull_request_template.md. :pr:`22665`
+* Add ``http-server.authentication.allow-forwarded-https`` configuration property to recognize X-Forwarded-Proto header. :pr:`22492`
+* Add ``node-scheduler.max-preferred-nodes`` configuration property to allow changing number of preferred nodes when soft affinity scheduling is enabled. :pr:`22562`
+* Add documentation for :func:`noisy_approx_set_sfm_from_index_and_zeros`. :pr:`22799`
+* Add documentation for noisy aggregate functions at :doc:`/functions/noisy`, including :func:`noisy_approx_distinct_sfm` and :func:`noisy_approx_set_sfm`. :pr:`22715`
```

Comment:
```suggestion
* Add documentation for noisy aggregate functions at :doc:`/functions/noisy`, including :func:`!noisy_approx_distinct_sfm` and :func:`!noisy_approx_set_sfm`. :pr:`22715`
```
Suppressing warning generated in doc build, see https://github.com/prestodb/presto/pull/23023 for explanation.